### PR TITLE
Changes done

### PR DESCRIPTION
**Pull request by rajdama**

### CODE REVIEW

**Comment by redxzeta** [comment]
u just need to push changes to the same branch that you worked on from the previous pull request

**Comment by redxzeta** [comment]
push the changes to your branch `changes`

**Comment by rajdama** [comment]
> push the changes to your branch `changes`
> no need to make a new pull request

I am getting error in my git bash that Changes cannot be resolved to branch

**Comment by rajdama** [comment]
so that's why I made new branch

**Comment by redxzeta** [comment]
whats the error? You could also use GitHub Desktop to make it easier

**Comment by rajdama** [comment]
> whats the error? You could also use GitHub Desktop to make it easier

"Changes cannot be resolved to branch this is the" error

**Comment by rajdama** [comment]
I will try Github desktop

**Comment by rajdama** [comment]
Now I got it what I have to do , can I make new pr in this branch i.e changes_done because a the branch Changes is not visible in git hub desktop
Screenshot 2021-06-01 192817

File: src/components/layout/NavigationBar.js [hunk 1]
```diff
@@ -12,7 +14,10 @@ const PetTypes = () =>
 
 export default function NavigationBar() {
   return (
+
+
```
Comment (redxzeta) [inline src/components/layout/NavigationBar.js:18]: can you remove the extra spaces

File: src/components/layout/NavigationBar.js [hunk 2]
```diff
@@ -33,12 +38,14 @@ export default function NavigationBar() {
             <Nav.Link as={Link} to="/resources">
               Resources
             </Nav.Link>
+
```
Comment (redxzeta) [inline src/components/layout/NavigationBar.js:41]: Remove Extra spaces

File: src/components/pets/PetType.js [hunk 3]
```diff
@@ -209,7 +209,7 @@ export default function PetType({ token }) {
             // array empty or does not exist
             return (
               <Col md={4} xs={12} key={pet.id} className="petList__column">
-                <Card style={{ width: "100%", paddingTop: 10 }}>
+                <Card style={{ width: "100%", paddingTop: 10 }} >
```
Comment (redxzeta) [inline src/components/pets/PetType.js:212]: remove the inline style so it only shows `<Card>`

**Review by redxzeta — CHANGES_REQUESTED** [review]
Alright just push the changes to this branch

**Comment by rajdama** [comment]
Now I am able to commit on this same branch thankyou for suggesting about GitHub Desktop :)

**Review by redxzeta — APPROVED** [review]

**Comment by redxzeta** [comment]
alright looks good to go :)

**Comment by rajdama** [comment]
Thankyou so much @redxzeta you really helped me a lot as I am a beginner. It was really a good experience working with you. Can I work with you in the future, if you have a team can I join ?

**Pull request merged**